next.config.js

```diff
@@ -30,6 +30,7 @@ const nextConfig = {
         REJECT_CONTRIBUTION: false,
         REJECTED_CATEGORIES: false,
         CHANGE_LOG_UPDATES_ENABLED: false,
+        CRYPTO_CONTRIBUTIONS_ENABLED: false,
```

Comment:
Not used.

Reply:
Ops, my mistake. I somehow added this again. I've corrected it. 👍🏽